src/screens/HomeScreen.tsx

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable react-native/no-inline-styles */
+import React from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+
+function HomeScreen() {
+  return (
+    <View style={styles.containerStyle}>
+      <Text style={{ fontSize: 50, marginTop: 60 }}>HomeScreen</Text>
```

Comment:
Can you please move inline style recommended seperate styles.

Reply:
okay sure, let me do.